### ISSUE TITLE
duplicates of symbols showing up

### ISSUE THREAD

**Issue by martinlippert** [spring-projects/sts4#1056]
After installing the latest 4.19.0 CI version into an Eclipse 2023-06 RC1 build and creating a Rest Service sample, I can see duplicated symbol information again. When invoking Cmd-6 on the editor, I see each symbol in the "symbols in workspace" area duplicated four times. The same happens for "symbols in project". It does NOT happen for "symbols in file".

**Comment by martinlippert** [comment]
The data in the persistent cache shows the symbols only once.

**Comment by martinlippert** [comment]
Restarting the IDE solves the issue, hard to reproduce.

**Comment by martinlippert** [comment]
Fixed with https://github.com/spring-projects/sts4/commit/3b82f4bb2f31bb822c21d8fbb6042e5ef362ed2b
